packages/animations/test/modal_test.dart

```diff
@@ -456,8 +458,89 @@ void main() {
       expect(find.byKey(topKey), findsNothing);
     },
   );
+
+  testWidgets(
+    'showModal builds a new route with specified route settings',
+    (WidgetTester tester) async {
+      const RouteSettings routeSettings = RouteSettings(
+        name: 'route-name',
+        arguments: 'arguments',
+      );
+
+      final Widget button = Builder(builder: (BuildContext context) {
+        return Center(
+          child: ElevatedButton(
+            onPressed: () {
+              showModal<void>(
+                context: context,
+                configuration: _TestModalConfiguration(),
+                routeSettings: routeSettings,
+                builder: (BuildContext context) {
+                  return const _FlutterLogoModal();
+                },
+              );
+            },
+            child: const Icon(Icons.add),
+          ),
+        );
+      });
+
+      await tester.pumpWidget(_boilerplate(button));
+      await tester.tap(find.byType(ElevatedButton));
+      await tester.pumpAndSettle();
+
+      // New route containing _FlutterLogoModal is present.
+      expect(find.byType(_FlutterLogoModal), findsOneWidget);
+
+      // Expect the last route pushed to the navigator to contain RouteSettings
+      // equal to the RouteSettings passed to showModal
+      final ModalRoute<dynamic> modalRoute = ModalRoute.of(
+        tester.element(find.byType(_FlutterLogoModal)),
+      )!;
+      expect(modalRoute.settings, routeSettings);
+    },
+  );
+  testWidgets(
```

Comment:
small nit: just an extra line in between the two tests.
```suggestion

  testWidgets(
```